src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -111,8 +111,18 @@
 Application Source Code was disclosed by the web server
 
 <H2>Strict Transport Security Header Scanner</H2>
-This scanner checks HTTPS response headers for the presence of a HTTP Strict Transport Security header (alerts if it is not found).
-Also checks if the HTTP Strict Transport Security (HSTS) header contains a max-age directive with value 0, which instructs clients to reset the entire HSTS Policy associated with the HSTS Host.
+This scanner checks HTTPS response headers for the presence of a HTTP Strict Transport Security (HSTS) header and tests for various implementation concerns, 
```

Comment:
The "checks HTTPS response headers" should be changed? It also checks HTTP and the body.